appendix.Rmd

```diff
@@ -77,4 +83,15 @@ knitr::write_bib(pkgs, file="packages.bib")
 knitr::kable(pkgs_df)
 ```
 
+These can be installed as follows:
+
+```{r, eval=FALSE}
+u = "https://git.io/vX8aa"
```

Comment:
Because installing that didn't seem to install all the deps. Also provides a way for users to see quickly what we're asking them to install. It's more explicit this way.